src/connected/autobinding.ts

```diff
@@ -156,4 +165,161 @@ export class AutoBindingService {
+      this.promptToAutoBindSingleOption(bestHit, connectionToBestHits, unboundFolder);
+    } else {
+      this.promptToAutoBindMultiChoice(connectionToBestHits, unboundFolder);
+    }
+  }
+
+  private async promptToAutoBindSingleOption(bestHit: MatchHit, connectionToBestHits: Map<BaseConnection,
+    MatchHit[]>, unboundFolder: VSCode.WorkspaceFolder) {
+    const [connection] = connectionToBestHits.keys();
+    const connectionName = getDisplayName(connection);
+    const result = await VSCode.window.showInformationMessage(
+      `There is a project to bind on ${connectionName}. Do you want to bind?`, 'Bind');
+    if (result === 'Bind') {
+      this.bindingService.saveBinding(bestHit.projectKey, connection.connectionId, unboundFolder);
+      VSCode.window.showInformationMessage(`Workspace folder '${unboundFolder.name}/'
+                      has been bound with project '${bestHit.projectKey}'`);
+    }
+  }
+
+  private async promptToAutoBindMultiChoice(connectionToBestHits: Map<BaseConnection, MatchHit[]>,
+                                            unboundFolder: VSCode.WorkspaceFolder) {
+    const result = await VSCode.window.showInformationMessage(
+      `There are projects chosen to bind on Sonar server(s). Do you want to bind?`, 'Bind');
+    if (result === 'Bind') {
+      const remoteProjectsQuickPick = VSCode.window.createQuickPick();
+      const folderName = unboundFolder.name;
+      remoteProjectsQuickPick.title = `Select Project to Bind with '${folderName}/'`;
+      remoteProjectsQuickPick.placeholder = `Select the remote project you want to bind with 
+      '${folderName}/' folder`;
+      remoteProjectsQuickPick.items = this.getQuickPickItemsToAutoBind(connectionToBestHits);
+      remoteProjectsQuickPick.ignoreFocusOut = true;
+      remoteProjectsQuickPick.onDidChangeSelection(selection => {
+        const projectToBind = selection[0];
+        const [connectionId] = projectToBind.description.split(' - ');
+        this.bindingService.saveBinding(projectToBind.description, connectionId, unboundFolder);
+        VSCode.window.showInformationMessage(`Workspace folder '${folderName}/'
+                      has been bound with project '${projectToBind.description} on ${connectionId} server'`);
+        remoteProjectsQuickPick.dispose();
+      });
+      remoteProjectsQuickPick.onDidTriggerItemButton(async e => {
+        const [connectionId, projectKey] = e.item.label.split(' - ');
+        const connection = Array.from(connectionToBestHits.keys()).find(c => c.connectionId === connectionId);
+        const serverType = getServerType(connection);
+        const baseServerUrl = this.bindingService.getBaseServerUrl(connection.connectionId, serverType);
+        remoteProjectsQuickPick.busy = true;
+        VSCode.commands.executeCommand(
+          Commands.OPEN_BROWSER,
+          VSCode.Uri.parse(`${baseServerUrl}?id=${projectKey}`)
+        );
+      });
+      remoteProjectsQuickPick.show();
```

Comment:
Let's maybetry during merge to directly call existing `createOrEditBinding()` method instead of duplicating the code